config/default/kustomization.yaml

```diff
@@ -33,10 +30,12 @@ patches:
 # 'CERTMANAGER' needs to be enabled to use ca injection
 # the following config is for teaching kustomize how to do var substitution
 - path: webhookcainjection_patch.yaml
+- path: namespace_delete.yaml
 
 resources:
 - ../manager
 - ../certmanager
+- namespace_create.yaml
```

Comment:
My understanding here is that we first delete the `system` namespace, and then create the `kcp-system` namespace. For my understanding, why are we doing this now and have not done this before?

Reply:
because previously we had the system namespace, and in the kustomization we had the global namespace transformer so it renames the system to scp-system. With the new namespace transformer, it only sets the resources namespaces not the namespace name itself.